.circleci/config.yml

```diff
@@ -6,13 +6,21 @@ defaults:
     - ubuntu
 
 commands:
-  abort_for_docs:
+
+  early_returns:
     steps:
       - run:
-          name: Avoid tests for docs
+          name: Early return if this is a docs build
+          command: |
+            if [[ $CIRCLE_BRANCH == *docs ]]; then
+              echo "Identifies as documents PR, no testing required."
+              circleci step halt
+            fi
+      - run:
+          name: Early return if this branch should ignore CI
           command: |
-            if [[ $CIRCLE_BRANCH == *_docs ]]; then
-              echo "Identifies as documents PR, no testing required"
+            if [[ $CIRCLE_BRANCH == *noci ]]; then
```

Comment:
Why not `_noci`? Isn't it more nice to have `yadayada_noci` than `yadayadanoci`?

Reply:
I have the Python habbit of using underscores as delineating. There's no good reason other than personal style. Though, I like a delineation. Feel free to pick between underscore and hyphen.

Reply:
I care for consistency. Let's use an underscore now (due to `_docs`) and decide on the global convention once the generator is implemented.

Reply:
@rafie Restored in latest push. I brought back the underscore.